<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reg</class>
 <widget class="QMainWindow" name="Reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="left_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="nickName_text">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>昵称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pwd1_text">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pwd2_text">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>确认密码</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gender">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>161</width>
         <height>39</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="female_rb">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>41</width>
          <height>19</height>
         </rect>
        </property>
        <property name="toolTipDuration">
         <number>2</number>
        </property>
        <property name="text">
         <string>女</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="male_rb">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>41</width>
          <height>19</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="text">
         <string>男</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="birthday">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>186</width>
         <height>42</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QSpinBox" name="day">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>46</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string>日</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="year">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string>年</string>
        </property>
        <property name="minimum">
         <number>1900</number>
        </property>
        <property name="maximum">
         <number>2020</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="month">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>46</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string>月</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="birthPlace_text">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>出生地</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="livingPlace_text">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>现居地</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reg_btn">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_btn">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="comment_text">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>331</width>
         <height>341</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;说明框&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
